2015/music-and-measure-theory/german/sentence_translations.json

```
@@ -506,7 +506,7 @@
  {
   "input": "Brief side note here, I had trouble deciding on how to illustrate small intervals, since if I scale the parentheses with the interval, you won't be able to see them at all, but if I just push the parentheses together, they cross over in a way that's potentially confusing.",
   "model": "nmt",
-  "translatedText": "Kurze Randbemerkung hier: Ich hatte Schwierigkeiten, mich zu entscheiden, wie kleine Intervalle dargestellt werden sollen, denn wenn ich die Klammern mit dem Intervall skaliere, werden Sie sie überhaupt nicht sehen können, aber wenn ich die Klammern einfach zusammenschiebe, kreuzen sie sich auf eine Weise, die möglicherweise verwirrend ist.",
+  "translatedText": "Kurze Randbemerkung hier: Ich hatte Schwierigkeiten, mich zu entscheiden, wie kleine Intervalle dargestellt werden sollen, denn wenn ich die Klammern mit dem Intervall skaliere, wirst du sie gar nicht sehen können, aber wenn ich die Klammern einfach zusammenschiebe, kreuzen sie sich auf eine möglicherweise verwirrende Art.",
```

No call to action to the viewer here, better use the neuter

```suggestion
  "translatedText": "Kurze Randbemerkung hier: Ich hatte Schwierigkeiten, mich zu entscheiden, wie kleine Intervalle dargestellt werden sollen, denn wenn ich die Klammern mit dem Intervall skaliere, wird man sie gar nicht sehen können, andererseits wenn ich die Klammern einfach zusammenschiebe, kreuzen sie sich auf eine möglicherweise verwirrende Art.",
```